From: Petr Tesarik <ptesarik@suse.com>
Subject: [PATCH] Define PN_XNUM if missing
Upstream: never, build fix for old distros

Older elfutils did not define this constant.

Signed-off-by: Petr Tesarik <ptesarik@suse.com>
Signed-off-by: Alexander Egorenkov <egorenar-dev@posteo.net>
---
 elf_info.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/elf_info.h
+++ b/elf_info.h
@@ -19,6 +19,10 @@
 #include <elf.h>
 #include <sys/types.h>
 
+#ifndef PN_XNUM
+#define PN_XNUM		0xffff
+#endif
+
 #define KEXEC_CORE_NOTE_NAME "CORE"
 #define KEXEC_CORE_NOTE_NAME_BYTES sizeof(KEXEC_CORE_NOTE_NAME)
 
